WPMedia/fusion-news-theme-blocks
# TMEDIA-106/Main Content Link

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description
This implements the `Skip To main` section link for arcSites so keyboard users can traverse to the Main section of the site, similar to a screen reader. Notes about the styling of this particular link can be found in the ticket: https://arcpublishing.atlassian.net/browse/TMEDIA-106

## Jira Ticket
- [TMEDIA-106](https://arcpublishing.atlassian.net/browse/TMEDIA-106)

## Acceptance Criteria
`Skip To main` link implemented and user is able to tab to that link and skip to the Main section of the webpage.

## Test Steps
- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout TMEDIA-106/main_content_link`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/default-output-block,@wpmedia/right-rail-block,@wpmedia/right-rail-advanced-block`
3. Go to http://localhost/pf/allblockspr/?_website=the-gazette
4. Hit tab once and observe the `Skip to Main` link appearing
5. Hit `return` and observe the page move to the `Main` section(any screen reader should match this)

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
